tests/run-make/raw-dylib-elf-verbatim-absolute/rmake.rs

```diff
@@ -0,0 +1,15 @@
+//@ only-elf
```

Comment:
The `x86_64-fortanix-unknown-sgx` target doesn't support dynamic linking. Can you please add `//@ ignore-sgx` to ignore this platform?

Reply:
Would `//@ needs-dynamic-linking` work too?

Reply:
Oh I didn't know about the `needs-dynamic-linking` directive. Good to know!